File: app/src/main/java/com/app/rupiksha/models/WalletFetchUser.java
package com.app.rupiksha.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class WalletFetchUser
{
    @SerializedName("username")
    @Expose
    private String username;
    @SerializedName("name")
    @Expose
    private String name;

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/app/rupiksha/models/WalletFetchUser.kt
package com.app.rupiksha.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class WalletFetchUser(
    @SerializedName("username")
    @Expose
    val username: String? = null,

    @SerializedName("name")
    @Expose
    val name: String? = null
)
